### PR TITLE
IW-1293 Adds Feeds Wiki Variables

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

**Review by latata — APPROVED** [review]

File: extensions/wikia/FeedsAndPosts/WikiVariables.class.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+class WikiVariables {
+	public function get() {
+		$cacheTTL = 60; // a minute
+
+		return \WikiaDataAccess::cache(
+			wfMemcKey( 'feeds-wiki-variables' ),
```
Comment (mszabo-wikia) [inline extensions/wikia/FeedsAndPosts/WikiVariables.class.php]: probably redundant, these are already memcached b/c of wikifactory

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: extensions/wikia/FeedsAndPosts/WikiVariables.class.php (hunk 1 above)
Comment (leshniak) [reply extensions/wikia/FeedsAndPosts/WikiVariables.class.php]: see changes, can we still get rid of memcache?

Comment (mszabo-wikia) [reply extensions/wikia/FeedsAndPosts/WikiVariables.class.php]: yep, all these variables are prefetched and cached by WFL at the start of the request

Comment (leshniak) [reply extensions/wikia/FeedsAndPosts/WikiVariables.class.php]: fixed

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/FeedsAndPosts/WikiVariables.class.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class WikiVariables {
+	public function get() {
+		global $wgServer, $wgDBname, $wgDisableHTTPSDowngrade, $wgEnableHTTPSForAnons;
+
+		$wikiVariables = [
+			'basePath' => $wgServer,
+			'dbName' => $wgDBname,
+		];
+
```
Comment (themech) [inline extensions/wikia/FeedsAndPosts/WikiVariables.class.php:13]: Cannot we use the existing `MercuryApi::getWikiVariables` method? It has the basePath, dbName and all the rest: https://witcher.fandom.com/wikia.php?controller=MercuryApi&method=getWikiVariables

Comment (leshniak) [reply extensions/wikia/FeedsAndPosts/WikiVariables.class.php:13]: Actually, we can. We did this to minimize the payload for feeds.

**Pull request merged**